services/tracing/public/cpp/trace_event_agent.cc

```diff
@@ -146,6 +150,13 @@ void LegacyTraceEventAgent::StartTracing(const std::string& config,
   // latency.
   base::TimeDelta time_offset = TRACE_TIME_TICKS_NOW() - coordinator_time;
   TraceLog::GetInstance()->SetTimeOffset(time_offset);
+#endif
+#if defined(CASTANETS)
+  if (std::string("renderer") ==
+      base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("type")) {
```

Comment:
how about defining like this ? for more easy modifications in future.
constexpr const char kProcessTypeSwitch[] = "type";